README.md

```diff
@@ -94,6 +94,16 @@ Refer to [the documentation site](https://vaadin.com/docs) or visit <https://vaa
 
 We would absolutely love to get the community involved, and we welcome any form of contributions – comments and questions on different communication channels, issues and pull request in the repositories, and anything that you build and share using our platform. If unsure, feel free to post an issue in this repository and we'll help you.
 
+## Building this project
+
+This project builds vaadin-bom, vaadin-core and some other artifacts that combines parts into packages that help to get started with various modules. Vaadin bom is generated using an NPM script based on the data in versions.json file. To make a development build execute following commands before doing the typical *mvn install*:
```

Comment:
> and some other artifacts that combines parts

What if we rephrase it a bit?

Reply:
For instance, it can be `This project builds Vaadin Platform artifacts. One of them, Vaadin bom, is generated using...`